src/pages/CreatedReviewManagePage/index.tsx

```diff
@@ -1,25 +1,71 @@
 //생성한 리뷰 관리 페이지
-import { useSendReview } from '@/apis/hooks'
+import { Suspense } from 'react'
+import { useLocation } from 'react-router-dom'
+import { Header } from '@/components'
+import { useGetReviewQuestion } from '@/apis/hooks'
+import useActiveTab from '@/pages/CreatedReviewManagePage/hooks/useReviewTab'
+import {
+  Tabs,
+  AllResponseReviewByResponser,
+  AllResponseReviewByReceiver,
+} from './components'
+import { PATH } from './constants'
 
 const CreatedReviewManagePage = () => {
-  const { mutate: sendReview } = useSendReview()
+  const { pathname } = useLocation()
+  const reviewId = pathname.replace(`${PATH.REVIEW_MANAGEMENT}`, '')
 
-  const handleButtonClick = () => {
-    sendReview(
-      { surveyId: 1, subjectIdList: [1, 2, 3] },
-      {
-        onSuccess: ({ data }) => {
-          console.log(data)
-        },
-      },
-    )
+  const { data: getReviewQuestion } = useGetReviewQuestion({
+    id: reviewId,
+  }).data
+  const { activeTab, changeTab } = useActiveTab('responser')
+
+  const REVIEW_MANAGE_TAB_CONTENT = {
+    responser: (
+      <Suspense
+        fallback={
+          <div className="flex items-center justify-center">
+            <div className="spinner-simple"></div>
+          </div>
+        }
+      >
+        <AllResponseReviewByResponser surveyId={reviewId} />
+      </Suspense>
+    ),
+    receiver: (
+      <Suspense
+        fallback={
+          <div className="flex items-center justify-center">
+            <div className="spinner-simple"></div>
+          </div>
+        }
+      >
+        <AllResponseReviewByReceiver surveyId={reviewId} />
+      </Suspense>
+    ),
   }
 
   return (
-    <>
-      <h1>생성한 리뷰 관리 페이지</h1>
-      <button className="btn" onClick={handleButtonClick} />
-    </>
+    <div className="flex h-auto min-h-screen flex-col bg-main-ivory  text-black dark:bg-main-red-100 dark:text-white">
+      <div className="sticky top-0 z-10">
+        <Header />
+        <Tabs activeTab={activeTab} setActiveTab={changeTab} />
+      </div>
+      <div className="m-0 flex w-full justify-center p-5 md:p-10">
+        <div className="m-0 flex w-[550px] max-w-[550px] flex-col gap-6">
+          <h1 className="text-xl md:text-2xl">{getReviewQuestion.title}</h1>
+          <h2 className="mt-2.5 text-sm md:text-xl">
+            {getReviewQuestion.description}
+          </h2>
+          <div>{REVIEW_MANAGE_TAB_CONTENT[activeTab]}</div>
+          <div className="mt-[50px] flex w-full justify-end">
+            <button className="w-25 btn h-10 rounded bg-active-orange text-white dark:text-black">
+              설문 마감
+            </button>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
```suggestion
    <div className="flex h-auto min-h-screen flex-col bg-main-ivory text-black dark:bg-main-red-100 dark:text-white">
      <div className="sticky top-0 z-10">
        <Header />
        <Tabs activeTab={activeTab} setActiveTab={changeTab} />
      </div>
      <div className="mx-auto flex w-full max-w-[800px] flex-col px-5 py-7 md:p-10">
        <h1 className="text-xl md:text-2xl">{getReviewQuestion.title}</h1>
        <h2 className="mt-3 text-sm md:mt-4 md:text-xl">
          {getReviewQuestion.description}
        </h2>
        <div className="mt-7">{REVIEW_MANAGE_TAB_CONTENT[activeTab]}</div>

        <button className="btn fixed bottom-10 self-end rounded-md bg-active-orange text-white dark:text-black">
          설문 마감
        </button>
      </div>
    </div>
```
![sa](https://github.com/prgrms-web-devcourse/Team-12-ReviewRanger-FE/assets/97094709/afbf1aa1-4f44-43ac-bd17-d4900b9b2ada)

효중님 세로 스크롤도 안 생기고 버튼도 잘 위치하게끔 바꿔봤습니다!

Reply:
`UserList`와 `AllResponseReviewByReceiver` 파일에도 코멘트 달아놨어요!!